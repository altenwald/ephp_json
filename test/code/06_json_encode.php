<?php

$data = ["hi" => "hello", "bye" => "goodbye"];
var_dump(json_encode($data, JSON_PRETTY_PRINT));
