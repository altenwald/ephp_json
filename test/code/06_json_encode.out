string(43) "{
    "hi": "hello",
    "bye": "goodbye"
}"
